src/MensaWidget/MWApiWrapperTest.class.st

```diff
@@ -65,35 +65,49 @@ MWApiWrapperTest class >> validMensaId [
 
 {
 	#category : #tests,
-	#'squeak_changestamp' : 'Louis Christ 6/4/2024 17:22'
+	#'squeak_changestamp' : 'Max 6/18/2024 18:34'
 }
 MWApiWrapperTest >> testApiCallWithInvalidUrl [
 
     | responseWrapper errorOccurred |
 
     errorOccurred := false.
-    [ responseWrapper := MWApiWrapper getMealsFrom: self class invalidMensaId on: self class invalidDate.
-    ] on: NetworkError do: [ :ex |
-        errorOccurred := true ].
+    [responseWrapper := MWApiWrapper getMealsFrom: self class invalidMensaId on: self class 	invalidDate using: WebClient] on: NetworkError do: [:ex | errorOccurred := true].
     self assert: (responseWrapper = '' or: errorOccurred).
 
 ]
 
 {
 	#category : #tests,
-	#'squeak_changestamp' : 'Louis Christ 6/4/2024 17:22'
+	#'squeak_changestamp' : 'Max 6/18/2024 18:29'
 }
 MWApiWrapperTest >> testApiCallWithValidUrl [
 
     | responseWrapper responseApi errorOccurred |
 
     errorOccurred := false.
-    [ responseWrapper := MWApiWrapper getMealsFrom: self class validMensaId on: self class validDate.
+    [ responseWrapper := MWApiWrapper getMealsFrom: self class validMensaId on: self class validDate using: WebClient.
     responseApi := WebClient httpGet: self class validApiUrl.
     ] on: NetworkError do: [ :ex |
         errorOccurred := true ].
     self assert: ( errorOccurred or: [responseApi content = responseWrapper]).
```

Comment:
remove space after `[` and in front of `]`

Reply:
Wasn't that your ticket? But I guess you are right, when merging we have to use my method instead of yours with the fixed spaces